publish-extensions.js

```diff
@@ -134,9 +137,42 @@ const flags = [
         continue;
       }
 
+      if (msVersion) {
+        if (!extension.repository) {
+          if (extension.download) ()
+          throw new Error(`${extension.id}: repository not specified`);
+        }
+        const download = await getReleases.resolveFromRelease(extension.repository, extension.version, msVersion);
+        if (!download) {
+          throw new Error(`${extension.id}: failed to resolve from releases`);
+        }
+        extension.version = msVersion;
+        extension.download = download;
+      }
+
+      if (Boolean(process.env.DRY_RUN)) {
+        continue;
+      }
+
       let timeout;
-      await new Promise((resolve, reject) => {
-        const p = cp.spawn(process.execPath, ['publish-extension.js', JSON.stringify(extension)], {
+      await new Promise(async (resolve, reject) => {
```

Comment:
Please revert changes to this promise. We should go manually through failed dry run and adjust extensions.json